README.adoc

```diff
@@ -205,25 +205,25 @@ include::finish/backendServices/src/main/java/io/openliberty/guides/event/dao/Ev
 ----
 
 To use the entity manager at runtime, inject it into our CDI bean through the
-[hotspot=14 file=0]`@PersistenceContext` annotation. The entity manager interacts with the persistence context. 
+[hotspot=PersistenceContext file=0]`@PersistenceContext` annotation. The entity manager interacts with the persistence context. 
 Every `EntityManager` instance is associated with a persistence context. The persistence context 
 manages a set of entities and is aware of the different states that an entity can have.
 The persistence context synchronizes with the database when a transaction commits.
 
-The [hotspot=12-43 file=0]`EventDao` class has a method for each CRUD operation, so let's break them down:
+The [hotspot=EventDao file=0]`EventDao` class has a method for each CRUD operation, so let's break them down:
 
-* The [hotspot=17-19 file=0]`createEvent()` method persists an instance of the [hotspot=17-70 file=1]`Event` entity class to the data store by
-calling the [hotspot=18 file=0]`persist()` method on an `EntityManager` instance. The entity instance becomes managed
+* The [hotspot=createEvent file=0]`createEvent()` method persists an instance of the [hotspot=Event file=1]`Event` entity class to the data store by
```

Comment:
`Event` hotspot that references the whole Event class doesn't highlight the whole class:
![image](https://user-images.githubusercontent.com/26514093/66071687-96e85200-e521-11e9-80ca-4e5d8c971532.png)